changelog/add-7961-track-transactions-search

```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: add
+
+Measure reporting-related interactions on the Transactions page
```

Comment:
WDYT about a more specific message?

```suggestion
Track filtering interactions on the Transactions page.
```

Reply:
fixed in https://github.com/Automattic/woocommerce-payments/pull/8020/commits/7fc466f769175892cd051bfc3f3a0802801932ba . thank you